### PR TITLE
f-card-with-content@0.1.0 - Initial generated component

### PR DESCRIPTION
**Pull request by xander-marjoram**
### Added
- `f-card-with-content` generated component.

---

### Added
- Initial generated component

---

### Fixed
- Some whitespace issues in template files

### CODE REVIEW

**Review by simonsandersje — DISMISSED** [review]

File: packages/components/molecules/f-card-with-content/package.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    "lint:style": "vue-cli-service lint:style",
+    "test": "vue-cli-service test:unit",
+    "test-component:chrome": "cross-env-shell TEST_TYPE=component wdio ../../../../wdio-chrome.conf.js",
+    "test-a11y:chrome": "cross-env-shell TEST_TYPE=a11y wdio ../../../../wdio-chrome.conf.js"
+  },
+  "browserslist": [
+    "extends @justeat/browserslist-config-fozzie"
+  ],
+  "dependencies": {
+    "@justeat/f-services": "1.7.0"
```
Comment (oliversweb) [inline packages/components/molecules/f-card-with-content/package.json]: Will you be needing this?

File: packages/components/molecules/f-card-with-content/test-utils/component-objects/f-cardWithContent.component.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    }
+
+    load () {
+        super.load(this.component);
+    }
+
+    open (url) {
+        super.open(url);
+    }
+
```
Comment (oliversweb) [inline packages/components/molecules/f-card-with-content/test-utils/component-objects/f-cardWithContent.component.js]: I think these methods can be deleted as the `Page` contains the same methods

File: packages/components/molecules/f-card-with-content/test/accessibility/axe-accessibility.spec.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+const CardWithContent = require('../../test-utils/component-objects/f-cardWithContent.component');
+
+const cardWithContent = new CardWithContent();
+
+describe('Accessibility tests', () => {
+    beforeEach(() => {
+        cardWithContent.load();
+    });
+    it('a11y - should test f-cardWithContent component WCAG compliance', () => {
```
Comment (oliversweb) [inline packages/components/molecules/f-card-with-content/test/accessibility/axe-accessibility.spec.js:12]: nit: new line needed

File: packages/components/molecules/f-card-with-content/test-utils/component-objects/f-cardWithContent-selectors.js [hunk 4]
```diff
@@ -0,0 +1,2 @@
+// eslint-disable-next-line import/prefer-default-export
+export const COMPONENT = '[data-test-id="cardWithContent"]';
```
Comment (oliversweb) [inline packages/components/molecules/f-card-with-content/test-utils/component-objects/f-cardWithContent-selectors.js:2]: fyi - Accessing this constant fails when calling line _12_ below in file `f-cardWithContent.coponent.js`, so to overcome this issue I recently just embedded this tag into line _12_ below and deleted this file completely (+ removed the import statement on line _2_).

File: packages/components/molecules/f-card-with-content/package.json (hunk 1 above)
Comment (xander-marjoram) [reply packages/components/molecules/f-card-with-content/package.json]: Nope, shall remove

File: packages/components/molecules/f-card-with-content/test-utils/component-objects/f-cardWithContent-selectors.js (hunk 4 above)
Comment (ashleynolan) [reply packages/components/molecules/f-card-with-content/test-utils/component-objects/f-cardWithContent-selectors.js:2]: Should we raise this with the QE's? I'm guessing that this not working is an issue we may want to solve if this is the pattern they want to keep?

Comment (xander-marjoram) [reply packages/components/molecules/f-card-with-content/test-utils/component-objects/f-cardWithContent-selectors.js:2]: It seemed to work for me so I'll leave it as-is for now

File: packages/tools/generator-component/generators/app/templates/__package__.json [hunk 5]
```diff
@@ -7,7 +7,7 @@
   "maxBundleSize": "<%= bundlewatchMaxSize %>kB"<% } %>,
   "files": [
     "dist"<% if(config.needsComponentTests) { %>,
-    "test-utils"<% } %> <% if(config.needsTestingApiMocks) { %>,
```
Comment (xander-marjoram) [inline packages/tools/generator-component/generators/app/templates/__package__.json:10]: Removed a space in the middle, which comes out of the generator as trailing whitespace

File: packages/tools/generator-component/generators/app/templates/src/index.js [hunk 6]
```diff
@@ -33,9 +32,7 @@ if (GlobalVue) {
 }
 
 // To allow use as module (npm/webpack/etc.) export component
-export default <%= name.component %>;
-
-<% } else { %>
+export default <%= name.component %>;<% } else { %>
```
Comment (xander-marjoram) [inline packages/tools/generator-component/generators/app/templates/src/index.js:35]: Files generated from this template had something like 3 blank lines at the end of the file

**Review by simonsandersje — APPROVED** [review]

**Review by oliversweb — APPROVED** [review]

**Comment by dandel10n** [comment]
Would be good to have more description about the component, why this one is needed when we have f-card with ability to add any content inside, some screenshots showing what it will look like etc.

**Pull request merged**